### DISCUSSION TITLE
Domain: soltools.app "This dapp could be malicious."

### DISCUSSION THREAD

**Discussion by SolToolsApp** [phantom/docs#221] [Help & Support] [answered]
### Summary

How do I resolve this error, we're building a SPL token generator but getting "This dapp could be malicious. Do not proceed unless you are certain this is safe."

Even if we add a simple send function, still get the same transaction warning. 

### Example

https://soltools.app/create

### Steps to Reproduce

If you go to soltools.app, click create token, connect your wallet, put in your info and click upload metadata.

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hello @SolToolsApp,

The best way to address this is to write in to the email address mentioned in the warning: review@blowfish.xyz

**Reply by SolToolsApp** [reply]
They don't respond. So in that case who else can we contact?

**Reply by adamdelphantom** [reply, marked as answer]
@SolToolsApp I have asked them to follow up with you. Have you heard from them?

**Reply by aliapg2019** [reply]
> @SolToolsApp I have asked them to follow up with you. Have you heard from them?

I've encountered the same issue. My website is nofacemint.com, and I would like to know how to resolve it. Additionally, what should I write and send to review@blowfish.xyz to ensure they approve me? Please guide me.

**Comment by aliapg2019** [comment]
@adamdelphantom
I've encountered the same issue. My website is nofacemint.com, and I would like to know how to resolve it. Additionally, what should I write and send to [review@blowfish.xyz](mailto:review@blowfish.xyz) to ensure they approve me? Please guide me.

**Comment by BananaToolss** [comment]
I also encountered the same problem. My website is https://sol.bananatools.top/. How can I make my website not show malicious messages?

**Reply by Lucky-Lotto** [reply]
Hi, sorry to disturb you, I saw that you had solved this problem. Can you tell me how you solved it? Thanks

**Comment by adamdelphantom** [comment]
@aliapg2019 @BananaToolss Please reach out to review@blowfish.xyz with your domain and the warning you are receiving. They will respond with next steps.

**Comment by DecimalsIA** [comment]
Discussed in https://github.com/orgs/phantom/discussions/221

Originally posted by MinimemeApp on October 24, 2024

Summary

We're encountering an issue with our application at https://minimeme.io/sign-up. Phantom is displaying a false warning indicating: "This dapp could be malicious. Do not proceed unless you are certain this is safe." However, we assure you that the project is legitimate and completely safe.

This warning is misleading for our users and prevents them from trusting and utilizing our service properly. We kindly request that the domain https://minimeme.io/ be added to the whitelist so that users can interact without unnecessary interruptions.

This project was created by the well-known YouTuber, Dalas Review, as a professional, trustworthy initiative. Here is the link to his video demonstrating the project and its creation: https://www.youtube.com/watch?v=shMSZ93CEPU&t=412s.

Example

https://minimeme.io/sign-up

Steps to Reproduce

Go to https://minimeme.io/sign-up.

Click on the "Sign Up" button.

Connect your wallet.

Enter your details and submit the form.

Observe the warning from Phantom about potential malicious activity.

**Reply by adamdelphantom** [reply]
Hello @DecimalsIA please contact review@blowfish.xyz for help with this issue.

**Comment by Lucky-Lotto** [comment]
I also get this error

https://lucky-lotto.xyz/

<img width="345" alt="image" src="https://github.com/user-attachments/assets/b20eb3cd-5d48-4e68-8041-a6e7de465a14">

**Reply by adamdelphantom** [reply]
@Lucky-Lotto please contact review@blowfish.xyz

**Comment by DecimalsIA** [comment]
Hello i only was contact with blowfish team really they have good service
solved in fast time. Howere i am bussy to now i only recomend at you
contact with they

**Comment by roguex33** [comment]
We've sent an email to review@blowfish.xyz, please check the mailbox

https://pepx.fun/
https://www.pepx.fun/

**Comment by Matarai** [comment]
We've sent an email to [review@blowfish.xyz](mailto:review@blowfish.xyz), please check the mailbox   
https://nft.matar.ai/
https://matar.ai

**Comment by baekhyunee7** [comment]
We've sent an email to [review@blowfish.xyz](mailto:review@blowfish.xyz), please check the mailbox
https://www.cutagames.com/

**Comment by adamdelphantom** [comment]
All domain review discussions should continue with the review@phantom.com

**Reply by baskombrutal** [reply]
solarswap.fun How do I resolve this error?

**Reply by adamdelphantom** [reply]
@baskombrutal Please contact review@phantom.com
